packages/l/lact/package.yml

```diff
@@ -0,0 +1,24 @@
+name       : lact
+version    : 0.5.4
+release    : 1
+source     :
+    - https://github.com/ilya-zlobintsev/LACT/archive/refs/tags/v0.5.4.tar.gz : ce55f19dab2cb3616b404fc53feb381ee1e10e948bed65c7c33dc3b361685891
+homepage   : https://github.com/ilya-zlobintsev/LACT
+license    : MIT
+component  : system.utils
+summary    : Linux AMDGPU Configuration Tool 
+networking : yes
+description: |
+    This application allows you to control your AMD GPU on a Linux system.
+builddeps  :
+    - pkgconfig(gtk4)
+    - pkgconfig(hwdata)
```

Comment:
No need to add `pkgconfig(hwdata)`  to builddeps because it is part of `system.base` component.

Reply:
I see. I did not know that. I've rebuilt it without hwdata.